stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -24,106 +33,106 @@ class Worksheet(_WorkbookChild):
+    def add_chart(self, chart: Incomplete, anchor: Incomplete | None = ...) -> None: ...
+    def add_image(self, img: Incomplete, anchor: Incomplete | None = ...) -> None: ...
+    def add_table(self, table: Table) -> None: ...
     @property
-    def tables(self): ...
-    def add_pivot(self, pivot) -> None: ...
+    def tables(self) -> TableList: ...
+    def add_pivot(self, pivot: Incomplete) -> None: ...
     def merge_cells(
         self,
-        range_string: Incomplete | None = ...,
-        start_row: Incomplete | None = ...,
-        start_column: Incomplete | None = ...,
-        end_row: Incomplete | None = ...,
-        end_column: Incomplete | None = ...,
+        range_string: str | None = ...,
+        start_row: int | None = ...,
+        start_column: int | None = ...,
+        end_row: int | None = ...,
+        end_column: int | None = ...,
     ) -> None: ...
     @property
-    def merged_cell_ranges(self): ...
+    def merged_cell_ranges(self) -> list[CellRange]: ...
     def unmerge_cells(
         self,
-        range_string: Incomplete | None = ...,
-        start_row: Incomplete | None = ...,
-        start_column: Incomplete | None = ...,
-        end_row: Incomplete | None = ...,
-        end_column: Incomplete | None = ...,
+        range_string: str | None = ...,
+        start_row: int | None = ...,
+        start_column: int | None = ...,
+        end_row: int | None = ...,
+        end_column: int | None = ...,
     ) -> None: ...
-    def append(self, iterable) -> None: ...
-    def insert_rows(self, idx, amount: int = ...) -> None: ...
-    def insert_cols(self, idx, amount: int = ...) -> None: ...
-    def delete_rows(self, idx, amount: int = ...) -> None: ...
-    def delete_cols(self, idx, amount: int = ...) -> None: ...
-    def move_range(self, cell_range, rows: int = ..., cols: int = ..., translate: bool = ...) -> None: ...
-    @property
-    def print_title_rows(self): ...
+    def append(self, iterable: Iterable[Incomplete]) -> None: ...
+    def insert_rows(self, idx: int, amount: int = ...) -> None: ...
+    def insert_cols(self, idx: int, amount: int = ...) -> None: ...
+    def delete_rows(self, idx: int, amount: int = ...) -> None: ...
+    def delete_cols(self, idx: int, amount: int = ...) -> None: ...
+    def move_range(self, cell_range: CellRange | str, rows: int = ..., cols: int = ..., translate: bool = ...) -> None: ...
+    @property
+    def print_title_rows(self) -> str: ...
     @print_title_rows.setter
-    def print_title_rows(self, rows) -> None: ...
+    def print_title_rows(self, rows: str | None) -> None: ...
     @property
-    def print_title_cols(self): ...
+    def print_title_cols(self) -> str: ...
     @print_title_cols.setter
-    def print_title_cols(self, cols) -> None: ...
+    def print_title_cols(self, cols: str | None) -> None: ...
     @property
-    def print_titles(self): ...
+    def print_titles(self) -> str: ...
```

Comment:
This can return `None` if `self.print_title_rows` and `self.print_title_cols` are both `None`: https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.0/openpyxl/worksheet/worksheet.py#L866-871

```suggestion
    def print_titles(self) -> str | None: ...
```